monitoring/templates/cloudwatch_exporter.yml

```diff
@@ -0,0 +1,9 @@
+---
+region: eu-west-1
+metrics:
+ - aws_namespace: AWS/ELB
```

Comment:
I guess that these metrics have to be provided by the user no?
What happens if the user doesn't provide any metrics?